libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/PasscodeActivity.java

```diff
@@ -195,24 +291,21 @@ public void enableLogout(boolean b) {
         logoutEnabled = b;
     }
 
-    @Override
-    public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
-
-        // Processing the editor action only on key up to avoid sending events like pass code manager unlock twice.
-        if ( actionId ==  EditorInfo.IME_ACTION_GO ||
-                (event != null && event.getAction() == KeyEvent.ACTION_UP)) {
-            final String pc = entry.getText().toString();
-            if (pc.length() < getMinPasscodeLength()) {
-                error.setText(getMinLengthInstructions(getMinPasscodeLength()));
-                return true; // return true indicating we consumed the action.
-            }
-            return (pc.length() > 0 && onSubmit(pc));
-        } else {
-            return true;
-        }
+    /**
+     * Used for tests to allow biometric when the device is not set up
+     *
+     * @param b True - if biometric checks skipped, False - otherwise.
+     */
+    public void forceBiometric(boolean b) {
+        forceBiometric = b;
     }
 
     protected boolean onSubmit(String enteredPasscode) {
+        boolean showBiometricEnrollment = !passcodeManager.getBiometricEnabled() &&
```

Comment:
Could we rename these methods to `isBiometricEnabled()`, `isBiometricEnrollmentShown()`, and `isBiometricAllowed()`? `is` is typically more appropriate than `get` for methods that return `boolean`.